test/celldevs_grid_utils_test.cpp

```diff
@@ -0,0 +1,80 @@
+//
```

Comment:
I didn't look into coverage reports, but it looks like extremely small test for so much code changes.

Reply:
To be honest, I am not a C++ pro... I did a minimal test to ensure that the main functions work properly.
It would be nice to meet with you and discuss about these things. For example, when compiling the Cell-DEVS examples, I get a couple of warnings regarding destructors stuff, but I don't know how to deal with them. The output provided by them, as well as the step-by-step execution, appears to be OK

Reply:
Sure, we can schedule a call with Laouen too. Let's do that by email.